### ISSUE TITLE
Duplicated random fiber stats in qc_stats.txt

### ISSUE THREAD

**Issue by akjohnson** [fiberseq/fiberseq-qc#3]
The updated stats file now has two segments with the same stat title:

```
# Note: ***Random fiber stats***
Number(Fibers)=97
# Note: ***Random fiber stats***
Number(Fibers)=97
```

Because in https://github.com/fiberseq/fiberseq-qc/blob/main/src/runall-qc.tcsh the make-plot-rand-fibers.sh script gets run twice, but has no context to alter the title of the stat when it outputs its intermediate.stat.txt file and then when the files get combined there's no way to tell them apart from the stat file alone

**Comment by sjneph** [comment]
It's just two entries for two images (one for each).  The numbers sometimes are different.

**Comment by sjneph** [comment]
I could throw something in there to differentiate them further.

**Comment by sjneph** [comment]
Addressed in commit addb1e4d65a66e4010711a07fb81092ad1389b80
